INA-7891: Update Incident Management settings "information" page (#30979)
DataDog/documentation: content/en/service_management/incident_management/incident_settings/information.md

To convert an incident to private in Slack, use `/datadog incident private`.

## Incident deletion

_Default: disabled_

When incident deletion is enabled for an incident type, any user with **Incidents Write** can delete any incident of the incident type.

After you delete an incident, the incident no longer influences incident analytics, and no user can access it. Deleted incidents cannot be recovered.

## Override status timestamps

_Default: disabled_

When timestamp overrides are enabled in an incident type, any user with **Incidents Write** can define timestamp overrides in any incident of that incident type.

When enabled, you can define overrides for the `declared`, `detected`, and `resolved` timestamps on an incident. These overrides apply only in incident search and [analytics][3]. Overrides do not affect entries on the incident timeline. You can delete overrides.

## Test incidents

_Default: disabled_

When test incidents are enabled in an incident type, any user with **Incidents Write** can declare test incidents of the incident type.

> Test incidents are visually distinguished from other incidents. By default, test incidents do not by appear in incident search, execute automations, execute notification rules, or affect analytics. The declarer can opt into these functions during declaration.

Reviewer: Should we add the purple banner?

```suggestion
Test incidents are visually distinguished by a purple banner. By default, test incidents do not by appear in incident search, execute automations, execute notification rules, or affect analytics. The declarer can opt into these functions during declaration.
```
